sklearn/ensemble/_forest.py

```diff
@@ -1069,6 +1140,23 @@ class RandomForestClassifier(ForestClassifier):
 
         .. versionadded:: 0.22
 
+    feature_importances_type : {"impurity", "permutation"}, default="impurity"
+        The type of feature importance to calculate:
+
+        - If "impurity", then gets impurity-based feature importance.
+          The importance of a feature is computed as the (normalized)
+          total reduction of the criterion brought by that feature.  It is also
+          known as the Gini importance.
+        - If "permutation", then gets out-of-bag permutation importance.
+          The importance corresponds with the average decrease in r2_score
+          across all tree when a feature is permuted or shuffled.
+
+    scoring : string, callable, or None, default=None
```

Comment:
I would not expose this as mentioned above. It would only be a non-public attribute for the moment.